core/actions/_lib/runActionInstance.ts

```diff
@@ -218,8 +235,9 @@ export async function runActionInstance(args: RunActionInstanceArgs) {
 				actionInstanceId: args.actionInstanceId,
 				pubId: args.pubId,
 				userId: isActionUserInitiated ? args.userId : null,
-				status: "error" in result ? ActionRunStatus.failure : ActionRunStatus.success,
-				result,
+				result: { scheduled: `Action to be run immediately` },
+				// we are setting it to `scheduled` very briefly
+				status: ActionRunStatus.scheduled,
```

Comment:
Let me know if you have a better idea for this, but the idea is to first set the actionRun to `pending`, then set it to `success` or `error` after it resolves.

Feels kind of silly though.

Reply:
makes sense to me! "scheduled" seems accurate enough